src/OpenSage.Game.CodeGen/ScriptActionsGenerator.cs

```diff
@@ -33,7 +33,7 @@ public override void Execute(GeneratorExecutionContext context)
             sb.AppendLine("    }");
             sb.AppendLine("}");
 
-            context.AddSource("ScriptActions.Execution.generated.cs", sb.ToString());
+            context.AddSource("ScriptActions.Execution.g.cs", sb.ToString());
```

Comment:
Required since this seems to be the default extension that **Coverlet** recognizes. It then knows that this is generated by  source generator. See https://github.com/coverlet-coverage/coverlet/issues/1084#issuecomment-857549034